build_all/base-libraries/AzureIoTHub/README.md

```diff
@@ -35,7 +35,7 @@ You should have the following ready before beginning with any board:
 
 -   Install the Azure IoT C SDK libraries by one of two options:
 	1. Generate the Libraries by executing the [`make_sdk.cmd`](https://github.com/Azure/azure-iot-pal-arduino/blob/master/build_all/make_sdk.cmd) script within the `build_all` folder, with args as follows:
-		- The folder you want to copy the libraries into as arg 1, and the board you are using (`esp8266` or `esp32`) as arg 2
+		- The folder you want to copy the libraries into as arg 1
```

Comment:
if `esp8266` and `esp32` are no longer arguments needed it might be a good idea to add as a backlog item to clean up the `make_sdk.cmd` to rid the unneeded options and switches.

Reply:
Deprecating make_sdk.cmd instead, as duplicating work from a cross-platform script is silly.